src/main/java/edu/ucsd/sbrg/cobra/COBRAparser.java

```diff
@@ -1,9 +1,9 @@
 /**
  *
  */
-package edu.ucsd.sbrg.cobra;
+package main.java.edu.ucsd.sbrg.cobra;
```

Comment:
See above. The package is wrong.

Reply:
I have made the changes, I think these changes were done automatically by IDE.